driver/collector/postgres_collector.py

```diff
@@ -320,6 +320,29 @@
     i.indrelid, i.indexrelid;
 """
 
+# query digest id not included pre pg 14
+LONG_RUNNING_QUERY_NO_ID_SQL_TEMPLATE = """
+SELECT
+    pid, query_start, datid, datname, usename, state
+FROM
+    pg_stat_activity
+WHERE
+    now() - query_start > interval '{latency_threshold_min} minutes';
```

Comment:
Maybe we can calculate a timestamp like "earliest_time", and do:

`WHERE query_start > {earlist_time}` ?